lib/tests/mocha/unit/chunkers.spec.ts

```diff
@@ -267,27 +267,29 @@ describe('chunkers', () => {
     };
     it('all', async () => {
       const { fromUrl } = await import('../../../tdf3/src/utils/chunkers.js');
-      const c: Chunker = fromUrl(urlFor(path));
+      const c: Chunker = await fromUrl(urlFor(path));
       const all: Uint8Array = await c();
       expect(all).to.deep.equal(b);
       expect(Array.from(all)).to.deep.equal(r);
     });
     it('one', async () => {
       const { fromUrl } = await import('../../../tdf3/src/utils/chunkers.js');
-      const c: Chunker = fromUrl(urlFor(path));
+      const c: Chunker = await fromUrl(urlFor(path));
       const one: Uint8Array = await c(1, 2);
       expect(one).to.deep.equal(b.slice(1, 2));
       expect(Array.from(one)).to.deep.eq([1]);
     });
     it('negative one', async () => {
       const { fromUrl } = await import('../../../tdf3/src/utils/chunkers.js');
+      // @ts-ignore
```

Comment:
Please at least add comments here. You can also usually do something more specific.

Also current preferred behavior is to use `ts-expect-error` for test code, and avoid `ts-ignore` as much as possible for new code

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error